containers/iam-reana-sync/add_reana_users.py

```diff
@@ -0,0 +1,15 @@
+import json
+import os
+import subprocess
+
+client_id = os.environ['CLIENT_ID']
+client_secret  = os.environ['CLIENT_SECRET']  
+reana_admin_token = os.environ['REANA_ADMIN_TOKEN']
+print(reana_admin_token)
```

Comment:
I don't think we want to print the admin token

Reply:
It was for testing, it doesn't hurt printing it as you can test if the env variable is passed correctly. But as you point out, safer to hide it, so I took the line out.